hummingbot/strategy/pure_market_making/constant_size_sizing_delegate.pyx

```diff
@@ -37,17 +49,30 @@ cdef class ConstantSizeSizingDelegate(OrderSizingDelegate):
             bid_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
             ask_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
 
+        if (bid_order_size ==0):
+            self.logger().network("", f"Buy(bid) order size is less than minimum order size. Buy order will not be placed")
+
+        if (ask_order_size ==0):
+             self.logger().network("",f"Sell(ask) order size is less than minimum order size. Sell order will not be placed")
+
+
         for active_order in active_orders:
             if active_order.is_buy:
                 has_active_bid = True
             else:
                 has_active_ask = True
 
+        if (quote_asset_balance < pricing_proposal.buy_order_prices[0] * bid_order_size):
+            self.logger().network("", f"Not enough asset to place the required buy(bid) order. Check balances.")
+
+        if (base_asset_balance < ask_order_size):
+            self.logger().network("", f"Not enough asset to place the required sell(ask) order. Check balances.")
```

Comment:
This one is better since the user warning is actionable. But it's still missing the log message - which should pinpoint to why the order wasn't made.